### DISCUSSION TITLE
On install, there's an error message "This file is not secure" [Windows version]

### DISCUSSION THREAD

**Discussion by csfalcao** [thanhlmm/refi-app#51] [Ideas]
Both Edge and Windows will display an error message about the install file could harm your computer because it's from an untrusted source. In both cases, you have to go to "add an exception" or "details>run it anyways". I think those messages can be confusing or intimidating to inexperienced users. Unfortunately, I didn't get the screenshots to help, and I didn't I find any article or image on google. Latest Windows 10 version.

**Comment by thanhlmm** [comment]
Hi I know the situation you have. It is because the app needs to be signed with a Microsoft dev account to make sure the app is safe, however, the dev account and signing fee is about 200$ /year 😅

**Comment by csfalcao** [comment]
Pricey
